### PR TITLE
Updated about section and links

### PR DESCRIPTION
**Pull request by gdcodes**
Updated description of the society, added icon link for Discord to social media links and added links to important sections of the website for convenience. Future work could include creating a more detailed contact page to replace the contact section with information about the current board.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,21 +31,21 @@
...
-                  <a target="_blank" href="mailto:gutechsoc@gmail.com"><i class="fas fa-envelope"></i></a>
-              </li>
+            <li>
+              <a target="_blank" href="https://www.facebook.com/gutechsoc"><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://twitter.com/gutechsoc"><i class="fab fa-twitter"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://discord.gg/EGtY9n"><i class="fab fa-discord"></i></a>
```
Comment (David132639) [inline index.html:41]:
```suggestion
              <a target="_blank" href="https://discord.gg/YeemgDg"><i class="fab fa-discord"></i></a>
```

Comment (David132639) [reply index.html:41]: The original link comes up as "invite invalid" for me

File: index.html [hunk 2]
```diff
@@ -31,21 +31,21 @@
...
+            </li>
+            <li>
+              <a target="_blank" href="https://www.youtube.com/channel/UCbwnhXjLXAWxc6Nj_z-tlyQ/videos?view=57&flow=grid"><i class="fab fa-youtube"></i></a>
+            </li>
+            <li>
+              <a target="_blank" href="https://www.linkedin.com/company/glasgow-university-tech-society?trk=organization-update_share-update_actor-text"><i class="fab fa-linkedin"></i></i></a>
```
Comment (David132639) [inline index.html:47]:
```suggestion
              <a target="_blank" href="https://www.linkedin.com/company/glasgow-university-tech-society"><i class="fab fa-linkedin"></i></i></a>
```

Comment (David132639) [reply index.html:47]: The option seems unnessesary

File: index.html (hunk 1 above)
Comment (gdcodes) [reply index.html]: Thank you for pointing this out! It was working earlier but I think at some point I must have copied an old link. I have made the suggested change now.

File: index.html (hunk 2 above)
Comment (gdcodes) [reply index.html]: That's true. I have committed the suggested change now. Thank you for spotting this!

File: index.html (hunk 1 above)
Comment (gdcodes) [reply index.html]: Thank you for pointing this out! I think I used a link with an expiry of 24 hours (https://support.discordapp.com/hc/en-us/articles/208866998-Invites-101). I have committed the suggested change. Is the new link a 'Never' expire link with 'No limit' to the number of uses?

Comment (David132639) [reply index.html]: Yep

Comment (gdcodes) [reply index.html]: Thank you for pointing this out! I think I used a link with an expiry of 24 hours by mistake (https://support.discordapp.com/hc/en-us/articles/208866998-Invites-101). I have committed the suggested change.

**Pull request merged**